scripts/cluster

```diff
@@ -1,28 +1,30 @@
 #!/usr/bin/env bash
 
 function initIbftConsensus() {
-    echo "Running with ibft consensus"
-    ./polygon-edge secrets init --insecure --data-dir test-chain- --num 4
+  echo "Running with ibft consensus"
+  ./polygon-edge secrets init --insecure --data-dir test-chain- --num 4
 
-    node1_id=$(./polygon-edge secrets output --data-dir test-chain-1 | grep Node | head -n 1 | awk -F ' ' '{print $4}')
-    node2_id=$(./polygon-edge secrets output --data-dir test-chain-2 | grep Node | head -n 1 | awk -F ' ' '{print $4}')
+  node1_id=$(./polygon-edge secrets output --data-dir test-chain-1 | grep Node | head -n 1 | awk -F ' ' '{print $4}')
+  node2_id=$(./polygon-edge secrets output --data-dir test-chain-2 | grep Node | head -n 1 | awk -F ' ' '{print $4}')
 
-    genesis_params="--consensus ibft --ibft-validators-prefix-path test-chain- \
+  genesis_params="--consensus ibft --ibft-validators-prefix-path test-chain- \
     --bootnode /ip4/127.0.0.1/tcp/30301/p2p/$node1_id \
     --bootnode /ip4/127.0.0.1/tcp/30302/p2p/$node2_id"
 }
 
 function initPolybftConsensus() {
-    echo "Running with polybft consensus"
-    genesis_params="--consensus polybft"
-    ./polygon-edge polybft-secrets --insecure --data-dir test-chain- --num 4
+  echo "Running with polybft consensus"
+  genesis_params="--consensus polybft"
+  ./polygon-edge polybft-secrets --insecure --data-dir test-chain- --num 4
 }
 
 function createGenesis() {
-    ./polygon-edge genesis $genesis_params \
-      --block-gas-limit 10000000 \
-      --premine 0x85da99c8a7c2c95964c8efd687e95e632fc533d6:1000000000000000000000 \
-      --epoch-size 10
+  ./polygon-edge genesis $genesis_params \
+    --block-gas-limit 10000000 \
+    --premine 0x85da99c8a7c2c95964c8efd687e95e632fc533d6:1000000000000000000000 \
+    --premine 0x0000000000000000000000000000000000000000 \
+    --epoch-size 10 \
+    --burn-contract 0:0x0000000000000000000000000000000000000000
```

Comment:
FYI @0xPolygon/core-edge, only these lines contain de-facto change. Other than that is applied auto formatting.

Reply:
As I can see, this is the only line added: 
`--burn-contract 0:0x0000000000000000000000000000000000000000`  ?

Reply:
And premine `0x0` address